### PR TITLE
Dashboard modules 4 in 1

### PR DESCRIPTION
**Pull request by agudek**
Added 6 new modules in 1 on the dashboard. These use real data.  The prebody field is added to the module definition so that actions can be performed on modules that require data before this data is received. This only works for modules that have customContainer set to true.

### CODE REVIEW

**Comment by lvdoorn** [comment]
Looks good! Maybe we can add some tests to keep up the coverage levels.

**Comment by agudek** [comment]
@lvdoorn Working on tests

File: spec/modules/dashboard-m4-modules_test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+define(["src/modules/dashboard-m4-modules"], function (module) {
+	var data = [{ "sessions":
```
Comment (borek2) [inline spec/modules/dashboard-m4-modules_test.js]: You might want to remove this because the usage of real data? Or is it still used?

Comment (borek2) [inline spec/modules/dashboard-m4-modules_test.js]: While i was typing this... i am stupid... this is a test NVM 💃

File: src/modules/dashboard-m4-modules.js [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+        svg.append("text")
+            .attr("x","420")
+            .attr("y","310")
+            .style("font-size","1.6em")
+            .style("font-weight","500")
+            .style("fill","rgb(228, 74, 74)")
+            .text(c);  
+    }
+
+    return {
```
Comment (borek2) [inline src/modules/dashboard-m4-modules.js]: I really like the setup of the object 👍

**Pull request merged**